website/content/api-docs/secret/consul.mdx

```diff
@@ -70,18 +70,20 @@ $ curl \
 
 This endpoint creates or updates the Consul role definition. If the role does
 not exist, it will be created. If the role already exists, it will receive
-updated attributes.
+updated attributes. At least one of either `consul_policies`, `consul_roles`,
+`service_identities`, or `node_identities` are required depending on the
+Consul version.
 
 | Method | Path                  |
 | :----- | :-------------------- |
 | `POST` | `/consul/roles/:name` |
 
 ### Parameters for Consul versions 1.11 and above
 
-- `partition` `(string: "")` <EnterpriseAlert inline /> - Specifies the Consul admin partition the token
-  will be generated within. The partition must exist, and the policies or roles assigned to the
-  Vault role must also exist inside the given partition. If not provided, the "default"
-  partition is used.
+- `partition` `(string: "")` - Specifies the Consul admin partition the token will be
+  generated within. The partition must exist, and the Consul policies or roles assigned to the
```

Comment:
```suggestion
 The partition must exist, and the Consul policies or roles assigned to the
```

Reply:
Same comment as above.

You can also say: Specifies the Consul admin partition in which the token is generated from within.

It may be clearer if we add the word 'from' to precede the word 'within'.